### PR TITLE
SW-2322 Pick colors for the bar chart from base color 300s

### PR DESCRIPTION
**Pull request by constanzauanini**
Screenshot 2022-11-29 at 11 16 03

### CODE REVIEW

File: src/utils/generateRandomColor.tsx [hunk 1]
```diff
@@ -9,4 +23,14 @@ const generateRandomColors = (numberOfColors: number) => {
   return colors;
 };
 
-export { generateRandomColor, generateRandomColors };
+const generateTerrawareRandomColors = (theme: Theme, numberOfColors: number) => {
+  const terrawareColorsList = terrawareColors(theme);
+  const colors = Array.from({ length: numberOfColors }).map((_, i) => {
+    const randomIndex = Math.floor(Math.random() * terrawareColorsList.length);
+    const selectedColor = terrawareColorsList.splice(randomIndex, 1);
+    return selectedColor[0] || generateRandomColor();
```
Comment (constanzauanini) [inline src/utils/generateRandomColor.tsx:31]: if numberOfColors is bigger than terrawareColors' length, we return a random color

Comment (karthikbtf) [reply src/utils/generateRandomColor.tsx:31]: We need to cycle through colors, no need to use random.
if there are 10 colors.. and 12 bars, we cycle through colors and repeat.
ie.
bar1 - color1
bar2 - color2
bar3 - color3...
bar10 - color10
bar11 - color1
bar 12 - color2

Comment (constanzauanini) [reply src/utils/generateRandomColor.tsx:31]: Oh, ok, but we will have repeated colors. Also between the two chart we will get same colors for the bars

Comment (karthikbtf) [reply src/utils/generateRandomColor.tsx:31]: That was my understanding from Rachel. We can check with her if needed.
In this current PR, is it possible to have 2 bars next to each other with the same color?

**Review by karthikbtf — APPROVED** [review]
Looks good otherwise, feel free to follow up with Rachel.

File: src/utils/generateRandomColor.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/utils/generateRandomColor.tsx:31]: It could be possible if generateRandomColor() returns same color twice, but low probability

Comment (karthikbtf) [reply src/utils/generateRandomColor.tsx:31]: Ok we can get this in maybe and ask Rachel in the Jira, follow up later if needed.

**Pull request merged**